pkg/infradb/utils.go

```diff
@@ -0,0 +1,294 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
apparently, the file should be named replay.go

Reply:
Maybe. I will chck it out

Reply:
Done